tokio/src/loom/std/atomic_u64_static_once_cell.rs

```diff
@@ -0,0 +1,36 @@
+use super::AtomicU64;
+use crate::loom::sync::{Mutex, atomic::Ordering};
+use crate::util::once_cell::OnceCell;
+
+pub(crate) struct StaticAtomicU64 {
+    init: u64,
```

Comment:
It is a bit annoying to store the initial value forever, but this is *only* for platforms that do not have `AtomicU64` **and** no `const fn Mutex::new()` (i.e. old versions of Rust), so it isn't a big deal IMO.